File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.js
// Last updated: 7/17/2026, 4:25:42 PM
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxWords = 0;

    for(let i=0;i<sentences.length;i++){
        let words = sentences[i].split(" ").length;

        if(words > maxWords){
            maxWords = words;
        }
    }
    return maxWords;
};

File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.cpp
// Last updated: 7/17/2026, 4:25:38 PM
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int maxWord = 0;

        for(int i = 0; i < sentences.size(); i++){
            int count = 1;

            for(int j = 0; j < sentences[i].size(); j++){
                if(sentences[i][j] == ' '){
                    count++;
                }
            }

            if(count > maxWord){
                maxWord = count;
            }
        }

        return maxWord;
    }
};